### ISSUE TITLE
statistics calculator (js beta) - step 54 Math.pow code check too strict

### ISSUE THREAD

**Issue by hbar1st** [freeCodeCamp/freeCodeCamp#55697]
### Describe the Issue

I think I ran into a bug in one of the statistics calculator project steps.
Basically, they're trying to get us to write a Math.pow call but the step doesn't accept the following code:
```js
 const standardDeviation = Math.pow(variance, 0.5);
```
But it does accept this instead:
```js
 const standardDeviation = Math.pow(variance, 1/2);
```
It is funny because the example in the step literally shows 0.5 being used (if via a variable).

### Affected Page

https://www.freecodecamp.org/learn/javascript-algorithms-and-data-structures-v8/learn-advanced-array-methods-by-building-a-statistics-calculator/step-54

### Your code

```
 const standardDeviation = Math.pow(variance, 0.5);

```

### Expected behavior

The step should accept any variable of 1/2 as the 2nd param of Math.pow (.5, 0.5 seem like valid variations of 1/2)

### System

N/A

**Comment by hbar1st** [comment]
Isn’t 0.5 the exact same as 1/2?
Also the instructions do not place backticks around the 1/2 so it is not clear at all that they actually meant that we should type exactly that? (Usually steps surround text that needs to be typed exactly in backticks which was not done here)

also, the example literally uses 0.5!!!

Please reopen for further discussion given all the above.
Thank you.

**Comment by Sembauke** [comment]
That's true. Sorry for the shortsightedness, we can update the example to 1/2?

**Comment by ojeytonwilliams** [comment]
Sure or simply accept both.

**Comment by Sembauke** [comment]
Lets do both then!
